Ola_alex.html

```diff
@@ -0,0 +1,78 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>
+      My First Website
+    </title>
+  </head>
+  <body itemscope itemtype="https://schema.org/Person"> //This can't be used for two people but we didn't tell you to only write for one person beforehand so this is our bad
+    <header>
+        <a href="#maincontent"> Skip to content</a>
+      <nav>
+          <a href="#">Home</a> |
+          <a href="#">Contact</a>
+      </nav>
+      
+    <h1><div itemProp="name">Name: Olamide & Alex</div> 
```

Comment:
The `itemProp` Should wrap the name, not ‘Name:’. A sematically meaningless span could markup the name with the itemProp on.

`<h1><div>Name: <span itemProp=“name”>Olamid & Alex</span></div></h1>